config/playground/helm/ml/helm-machinelearning.yaml

```diff
@@ -107,8 +107,8 @@ opensearchJavaOpts: "-Xmx2G -Xms2G"
 
 resources:
   requests:
-    cpu: "2"
-    memory: "16Gi"
+    cpu: "16"
```

Comment:
The node's max CPU and Memory as following. I don't think that we increase to that high. 

Instance Size | vCPU | Memory (GiB) | Instance Storage (GB) | Network Bandwidth (Gbps)*** | EBS Bandwidth (Mbps)
-- | -- | -- | -- | -- | --
m5.2xlarge | 8 | 32 | EBS-Only | Up to 10 | Up to 4,750

Reply:
I think in the quip doc we agree to upgrade the instance into `r5.4xlarge`?

Instance Size | vCPU | Memory (GiB) | Instance Storage (GB) | Network Bandwidth (Gbps)*** | EBS Bandwidth (Mbps)
-- | -- | -- | -- | -- | --
r5.4xlarge | 16 | 128 | EBS-Only | up to 10 | 4,750

Reply:
r5.4xlarge's max memory is 128 GBs, but you are asking 128 GBs.

Reply:
Oh I misunderstand the configure here, so we are only going to use 1 instance to create the cluster right?